### ISSUE TITLE
Training a new model

### ISSUE THREAD

**Issue by sidc9** [jwyang/face-alignment#23]
Hi,

Thank you for the code! It has been very useful to me. But I have run into a snag and I was hoping you can help.

I have been trying to train my own model with your code. I have a dataset of around 65000 images with  49 landmark points for each image.

While setting the parameters, I set "global_params.landmark_num" in InitializeGlobalParam() to 49. But the execution always crashes while loading the images at around the 16000 image mark or 30000 mark with the error - OpenCV Out of Memory exception.

However, if I leave the landmark_num to 68 the training completes successfully. But then if I try to Test the model or run the Demo mode, the program always crashes (which I think is what should happen since the global parameter claims there are 68 points but in reality there are only 49).

It would be really good if you could help me out a little.

**Comment by jwyang** [comment]
Hi, thanks for your interest.

So you are using the C++ project, right? Please give me some time to check
it. Because i suppose the C++ codes is not aimed to support different
number of loanmarks.

Best,

Jianwei

**Comment by sidc9** [comment]
Yes I'm using the C++ project. Thank you for the prompt response!

**Comment by jwyang** [comment]
ok, I see, I will help to figure out why it did not work after changing the
number of landmarks.

**Comment by sidc9** [comment]
Hi Jianwei,

I was wondering if you had any insights on the problem?

**Comment by jwyang** [comment]
I am very sorry that I am a little busy these days, I have not yet found the causes. Maybe you can also send email to Yuanxun Wang, his email is wyxswjtu@163.com. He is the author of C++ codes.

**Comment by jwyang** [comment]
@sidc9 I am very sorry that I am a little busy these days, I have not yet found the causes. Maybe you can also send email to Yuanxun Wang, his email is wyxswjtu@163.com. He is the author of C++ codes.
